_episodes/05-cond.md

```diff
@@ -147,7 +147,9 @@ seemed to rise like a straight line, one unit per day.
 We can check for this inside the `for` loop we wrote with the following conditional:
 
 ~~~
-if numpy.max(data, axis=0)[0] == 0 and numpy.max(data, axis=0)[20] == 20:
+max1 = numpy.max(data, axis=0)[0]
+max2 = numpy.max(data, axis=0)[20]
```

Comment:
I agree with @annefou that `max1` and `max2` are not descriptive enough names to make things less confusing. We could use more descriptive variable names identifying the significance of the values such as `max_day_0` and `max_day_20`.